### ISSUE TITLE
[MU4 Issue] Accessibility: keyboard focus remains in menu when adding text

### ISSUE THREAD

**Issue by MarcSabatella** [musescore/MuseScore#15979]
**Describe the bug**
If you use the keyboard to access the Add menu to add text to your score, keyboard focus remains in the menu, making it impossible to type letters that correspind to menu mnemonics (eg, "F" for "File")

**To Reproduce**
1. Select any note or rest
2. Press Alt+A to open the Add menu
3. Cursor down to Text
4. Press Right to open the Text submenu
5. Cursor down to Staff text
6. Press Enter to add a staff text
7. Start typing
8. See errror: you can't type the letters F, E, V, A, O, T, P, H, or D, because these keys open the corresponding menu (File, Edit, etc)

**Expected behavior**
Keyboard focus should be on the score so you can type normally

**Platform information**
 - OS: Linux

**Additional context**
Workaround is to press Esc to return focus to the score before you begin typing.  Orca at least does read the menu as it opens, giving you a clue that something has gone wrong.   Not sure if that's true for all screenreaders.

**Comment by jmwmusic** [comment]
@MarcSabatella could you retest this it seems to be working fine for me now with NVDA.

**Comment by MarcSabatella** [comment]
I can confirm it is now working as expected for me on Linux.  I don't see a specific PR that fixed it, but it's also possible it was fixed by a Qt update.
